src/components/GameBoard.jsx

```diff
@@ -38,6 +58,18 @@ function GameBoard({ initialGameData, initialUserState, socket, userName }) {
       <div>Current Round: {gameData.currentRound}</div>
       <div>Draw Timer: {gameData.drawTimer}</div>
       <div>Current Drawer: {gameData.currentDrawerName} </div>
+      <div> Score Board:  </div>
```

Comment:
looks good! could be a good place to add conditional styling depending on if theyve guessed the current word or not